app/src/main/java/in/tosc/digitaloceanapp/activities/DropletActivity.java

```diff
@@ -93,16 +94,19 @@ public void onClick(View view) {
         toggle.syncState();
         doClient.getAccount().enqueue(new Callback<AccountInfo>() {
             @Override
-            public void onResponse(Call<AccountInfo> call, Response<AccountInfo> response) {
-                String email = response.body().getAccount().getEmail();
-                if (((TextView) drawer.findViewById(R.id.accountEmail)) != null) {
-                    ((TextView) drawer.findViewById(R.id.accountEmail)).setText(email);
-                }
-                ImageView profilePic = ((ImageView) drawer.findViewById(R.id.accountPic));
-                if (profilePic != null && email != null && !email.isEmpty()) {
-                    Picasso.with(DropletActivity.this).load("https://www.gravatar.com/avatar/" + md5(email)).into(profilePic);
-                }
+            public void onResponse(@NonNull Call<AccountInfo> call, @NonNull Response<AccountInfo> response) {
+                String email = null;
+                if (response.isSuccessful() && response.body() != null) {
+                    email = response.body().getAccount().getEmail();
 
+                    if (((TextView) drawer.findViewById(R.id.accountEmail)) != null) {
+                        ((TextView) drawer.findViewById(R.id.accountEmail)).setText(email);
+                    }
+                    ImageView profilePic = ((ImageView) drawer.findViewById(R.id.accountPic));
```

Comment:
How come you wrap the assignment in parenthesis? Does this achieve anything extra? 🤔

Couldn't you simply do `ImageView profilePic = (ImageView) drawer.findViewById(R.id.accountPic);`?

Reply:
okay Ill remove the parenthesis